index.bs

```diff
@@ -169,15 +211,106 @@ This allows developers to detect support for paint timing for a particular [=bro
 Processing model {#sec-processing-model}
 ========================================
 
-Reporting paint timing {#sec-reporting-paint-timing}
+Associated Image Requests {#sec-associated-image-requests}
+----------------------------------------------------------
+
+Each {{Element}} has an <dfn>associated image request</dfn> which is an [=image
+request=] or null, initially null.
+
+When the processing model for an {{Element}} <em>element</em> of type
+{{HTMLImageElement}}, {{SVGImageElement}}, or {{HTMLVideoElement}} creates a
+new image resource (e.g., to be displayed as an image or poster image),
+<em>element</em>'s <a>associated image request</a> is set to the <a>image
+request</a> of the created image resource.
+
+Note: Every image resource that is obtained from a URL whose
+<a spec=url>scheme</a> is equal to "data" has an associated <a>image request</a>
+which is not fetched but still needs to be loaded. This request can be the
+<a>associated image request</a> of an {{Element}}.
+
+Note: The current language is vague since it does not point to specific
+algorithms. This can be made more rigorous when the corresponding processing
+models have a more unified processing model.
+
+Every {{Element}} has a list of <dfn>associated background image requests</dfn>
+which is initially an empty array. When the processing model for the {{Element}}
+<em>element</em>'s style requires a new image resource (to be displayed as
+background image), the <a>image request</a> created by the new resource is
+appended to <em>element</em>'s <a>associated background image requests</a>.
+
+NOTE: We assume that there is one <a>image request</a> for each {{Element}} that
```

Comment:
This wording is a bit awkward. Perhaps "An {{Element}} can have several <a>image requests</a>, e.g. if its <a>background-image</a> property has multiple values.